demo/settings.yaml

```diff
@@ -27,6 +27,9 @@ black_list:
 # logging level options are: critical, error, warning, info, debug, notset
 log_level: 'info'
 
+# The oldest number of days a job can be. To disbable this option set value to -1
+threshold_days: 10
```

Comment:
Agreed! Went a little overboard with 10 days :sweat_smile: I think 20 might be reasonable, but won't change until we get some more input on this from the community.

Reply:
sounds like we've settled on 30?

Reply:
Because of some good points that @markkvdb we decided it might be best to make date_filter a command line option by passing `--max_listing_days`. It works very much like proxy in the sense that there is no "default" value. If `--max_listing_days` is passed without an argument(the number of days), validate.py invalidates it and gives an error to the user. Does this answer your question?